README.md

```diff
@@ -29,6 +29,20 @@ Hash.use_dot_syntax = true
 {name: 'Pat'}.name #=> 'Pat'
 ```
 
+By default HashDot does not respect defaults because this means any method would return a default of `nil` instead of raise a `NoMethodError1`. Defaults can be enabled either globally or per-instance.
```

Comment:
Typo: `NoMethodError1` -> `NoMethodError`.

Here's an alternate suggestion that I think is a little clearer:

```
By default HashDot returns `NoMethodError` when a hash key does not exist.  The value of `Hash#default` can be used instead, either globally or per-instance, via:
```

PS - Sorry I thought all you were going to do was bump the version number otherwise I'd have volunteered for the README change.